python/test/Ice/operations/TestI.py

```diff
@@ -510,26 +510,25 @@ def opStringLiterals(self, current: Ice.Current) -> Sequence[str]:
     def opWStringLiterals(self, current: Ice.Current) -> Sequence[str]:
         return self.opStringLiterals(current)
 
-    # TODO: Marshaled result
     def opMStruct1(self, current: Ice.Current):  # pyright: ignore
-        return Test.MyClass.OpMStruct1MarshaledResult(Test.Structure(), current)
+        return Test.Structure()
```

Comment:
You plan to fix the pyright ignore in a follow-up PR?

Reply:
I should remove that now!